packages/web-components/src/components/rux-segmented-button/rux-segmented-button.tsx

```diff
@@ -127,6 +127,10 @@ export class RuxSegmentedButton {
         target.closest('li')?.classList.remove('--focused')
     }
 
+    //this random number is appended to <input name="" in order to give each segmented button group a unique name
+    //this allows users to tab between button groups that are adjacent to each other
+    private random = Math.floor(Math.random() * 100000)
```

Comment:
Is there a chance that this same random number could be given to another name attr? I mean I think the chances are 1 in a 100000. I'm more asking out of curiosity lol